google/cloud/spanner_v1/session.py

```diff
@@ -319,9 +328,12 @@ def run_in_transaction(self, func, *args, **kw):
 
         :type kw: dict
         :param kw: (Optional) keyword arguments to be passed to ``func``.
-                   If passed, "timeout_secs" will be removed and used to
+                   If passed:
+                   "timeout_secs" will be removed and used to
                    override the default retry timeout which defines maximum timestamp
                    to continue retrying the transaction.
+                   "commit_request_options" will be removed and used to set the
+                   request_options which defines the common options for requests.
```

Comment:
nit: "commit_request_options" will be removed and used to set the request options for the commit request.